web-common/src/features/dashboards/state-managers/actions/dimension-filters.ts

```diff
@@ -11,6 +11,7 @@ export function toggleDimensionValueSelection(
   dimensionName: string,
   dimensionValue: string,
   keepPillVisible?: boolean,
+  uniqueValue?: boolean,
```

Comment:
This parameter is a little hard to understand on its own. Maybe `isExclusiveFilter` and add a little comment to explain it?